datajoint_utilities/dj_worker/__init__.py

```diff
@@ -153,28 +183,40 @@ def _clean_up(pipeline_modules, additional_error_patterns=[], stale_hours=24):
     + generic-type error jobs
     + stale "reserved" jobs
     """
-    _generic_errors = ["%Deadlock%", "%DuplicateError%", "%Lock wait timeout%",
-                       "%MaxRetryError%", "%KeyboardInterrupt%",
-                       "InternalError: (1205%", "%SIGTERM%",
-                       "LostConnectionError"]
+    _generic_errors = [
+        "%Deadlock%",
+        "%DuplicateError%",
+        "%Lock wait timeout%",
+        "%MaxRetryError%",
+        "%KeyboardInterrupt%",
+        "InternalError: (1205%",
+        "%SIGTERM%",
+        "LostConnectionError",
+    ]
 
     for pipeline_module in pipeline_modules:
         # clear generic error jobs
-        (pipeline_module.schema.jobs & 'status = "error"'
-         & [f'error_message LIKE "{e}"'
-            for e in _generic_errors + additional_error_patterns]).delete()
+        (
+            pipeline_module.schema.jobs
+            & 'status = "error"'
+            & [
+                f'error_message LIKE "{e}"'
+                for e in _generic_errors + additional_error_patterns
+            ]
+        ).delete()
         # clear stale "reserved" jobs
-        stale_jobs = ((pipeline_module.schema.jobs & 'status = "reserved"').proj(
-            elapsed_days='TIMESTAMPDIFF(HOUR, timestamp, NOW())')
-                      & f'elapsed_days > {stale_hours}')
+        stale_jobs = (pipeline_module.schema.jobs & 'status = "reserved"').proj(
```

Comment:
There is a better way to do this. The jobs table contains the connection number. A stale job is one for which there is no connection anymore.

Reply:
Also, you need to disable serialization when moving data.  You will save a lot of time and errors.  `dj.blob.bypass_serialization = True`.